infra/charts/feast-feature-server/values.yaml

```diff
@@ -15,6 +15,16 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+metrics:
+  enabled: false
+  otelCollector:
+    endpoint: http://{{ .Values.service.name }}-collector.{{ .Release.namespace }}.svc.cluster.local  # sample endpoint: "otel-collector.default.svc.cluster.local:4317"
+    port: 4317
+
+
+service:
```

Comment:
is this metrics related? if so, maybe it should live under that umbrella of values to help differentiate it? if not, at the very least the name of the value should better describe its use-case... maybe `otel_service`?